infrastructure/channels/serialize-data.js

```diff
@@ -0,0 +1,38 @@
+
+let cyclicArray = [2];
```

Comment:
This would seem more consistent with the theme that follows:

```suggestion
let cyclicArray = [1];
```